packages/ui-checkbox/src/CheckboxGroup/props.ts

```diff
@@ -35,14 +35,15 @@ import type { FormMessage } from '@instructure/ui-form-field'
 import type { PropValidators } from '@instructure/shared-types'
 
 import { Checkbox } from '../Checkbox'
+import type { CheckboxProps } from '../Checkbox/props'
 
 type CheckboxGroupOwnProps = {
-  children?: React.ReactNode // TODO: oneOf([Checkbox])
+  children?: React.ComponentElement<CheckboxProps, Checkbox>[]
```

Comment:
Sadly this is not a perfect substitute for 'ChildrenPropTypes.oneOf([Checkbox])' so let's leave the comment here until we find the best way to types this.